jhonny-woliver - AMARO

jhonnywoliver opened the pull request:
Teste para desenvolvedor frontend AMARO
Jhonny Woliver

jhonnywoliver committed:
jhonny-woliver - AMARO

Teste para desenvolvedor frontend AMARO
Jhonny Woliver

cassiocardoso commented:
Olá @jhonnywoliver recebemos sua solução. Em breve entraremos em contato com um feedback.

Obrigado por participar.

rodrigobacelli commented:
Oi @jhonnywoliver, nós estamos utilizando o Linux aqui para o desenvolvimento. Teria como compilar o projeto e atualizar a PR com um build que possamos testar aqui?

jhonnywoliver committed:
desafio front-end AMARO

desafio front-end AMARO
(inclusão da escolha por "tamanho" e dados salvos no session Storage)
Jhonny Woliver - 16/09/2016

jhonnywoliver committed:
Revert "desafio front-end AMARO"

This reverts commit ee8f7af73fc75cf2f1894e7e936ae8432ba9bdf7.

jhonnywoliver committed:
Desafio front-end Amaro - Jhonny Woliver

Teste para desenvolvedor front-end Amaro
